Gemfile

```diff
@@ -15,14 +15,9 @@ group :development, :test do
   gem "rspec"
   gem "rspec_junit_formatter" # used by CircleCI
   gem "rspec-rails"
-  # NOTE: the `standard` way does not support using the rubocop CLI with
-  # extensions like standard-rails, so until that changes or we decide to switch
-  # to the standardrb CLI (which would need CI and editor support), we have to
-  # carry both this dependency and standard-rails.
-  gem "rubocop-rails", require: false
```

Comment:
TIL that adding `standard-rails` to the `require` section of the rubocop config means we no longer have to carry this direct dependency. That's a minor win, at least.